fig.yml

```diff
@@ -0,0 +1,18 @@
+galera1:
+  build: .
+  command: --wsrep-cluster-address=gcomm://
```

Comment:
The option --wsrep-cluster-address just tells where to find the cluster aka meetingmembers.
Being the first one starting the meeting you don't join the meeting. Thats why there is no address to join.